modules/server/src/test/ts/compiler/ImportsTest.ts

```diff
@@ -0,0 +1,99 @@
+import { assert } from 'chai';
+import * as fc from 'fast-check';
+import { describe, it } from 'mocha';
+import { generateImports } from '../../../main/ts/bedrock/compiler/Imports';
+
+const validPolyfills = [ 'ArrayBuffer', 'Map', 'Object', 'Promise', 'Set', 'Symbol', 'TypedArray', 'WeakMap', 'WeakSet' ];
+
+const convertPolyfillName = (name: string) => {
```

Comment:
This seems to duplicate production code.

Reply:
I wonder if this whole test is a bit too much like the implementation. You may be a better off just using standard unit tests for this.

Reply:
Yeah, there's nothing testing that specific function atm, so should probably export that and unit test it. Also worth noting this isn't a duplicate, as the actual code determines the whole path while this only does the module name.

Reply:
Have added something to cover the polyfill name conversion as well. How does that look?